web-ng/src/app/components/feature-panel/feature-panel.component.html

```diff
@@ -42,17 +48,28 @@
       </mat-menu>
     </mat-card-header>
     <mat-card-content>
-        <div *ngFor="let field_entry of observation.form.fields ">
-          <label class="field-label">{{ field_entry[1].label?.get(lang) }}</label>
-          <div class="field-response" *ngIf="field_entry[1].type == fieldTypes.TEXT">
-            {{ observation.responses?.get(field_entry[1].id)?.value }}
-          </div>
-          <div class="field-response" *ngIf="field_entry[1].type == fieldTypes.MULTIPLE_CHOICE">
-            <span class="option-label" *ngFor="let option of observation.responses?.get(field_entry[1].id)?.value">
-              {{ option.label?.get(lang) }}
-            </span>
-          </div>
+      <div *ngFor="let field_entry of observation.form.fields">
```

Comment:
Out of scope of this PR, but while we're changing these lines anyway -  looks like we only use field_entry[1] , which contains the values in the `fields` map. Is there a way to just iterate over those instead? Maybe we can expose them as a property of the component?

Reply:
I tried to fix this directly in your PR, but it looks like I don't have write access to your branch; I'll send a follow-up PR once this is merged.